proto/self_serve/app/v1/app.proto

```diff
@@ -2,6 +2,16 @@ syntax = "proto3";
 
 package self_serve.app.v1;
 
+// Wrapper message for all messages that can be sent to and from the app. The reason we do this is to ensure static
+// typing in the clients while the server can still handle all messages through the Any type.
+message W {
+  oneof w {
+    StartCapture start_capture = 1;
+    RequestState request_state = 2;
+    AbortSignup abort_signup = 3;
```

Comment:
Can you add Ping and Pong while we're here?

Reply:
Check my message above. I think ping-pong should stay in a different functionality out of self_serve.